test_jpeg/test/test_rlecore.py

```diff
@@ -0,0 +1,130 @@
+from myhdl.conversion import *
+from myhdl import *
+from jpegenc.subblocks.RLE.RLECore.rlecore import DataStream, rle
+from jpegenc.subblocks.RLE.RLECore.rlecore import RLESymbols, RLEConfig
+from testcases import *
+
+red = 1
+green = 2
+blue = 3
+
+WIDTH_RAM_ADDRESS = 6
+WIDTH_RAM_DATA = 12
+
+
+def start_of_block(clock, start):
```

Comment:
Don't repeat code like this, put these in a separate python.module and import them into the tests.